core/tests/fireproof/attachable-subscription.test.ts

```
@@ -0,0 +1,500 @@
+      });
+
+      expect(totalSubscriptionFires).toBeGreaterThan(0);
+      console.log(`✅ Total subscription fires across all databases: ${totalSubscriptionFires}`);
+
+      // Verify data was synced correctly across all databases
+      await Promise.all(
+        dbs.map(async (db) => {
+          for (const key of keys) {
+            const doc = await db.db.get(key);
+            expect(doc._id).toBe(key);
+            expect((doc as any).value).toBe(key);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)